src/main/java/meeseeks/storage/Storage.java

```diff
@@ -0,0 +1,48 @@
+package meeseeks.storage;
+
+import meeseeks.parser.Parser;
+import meeseeks.task.TaskList;
+import meeseeks.task.Task;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+
+public class Storage {
+    private String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public ArrayList<Task> load() throws IOException {
+        Path path = Path.of(filePath);
+        if (!Files.exists(path)) {
+            return new ArrayList<>();
+        }
+
+        ArrayList<String> lines = new ArrayList<>(Files.readAllLines(path));
+        ArrayList<Task> tasks = new ArrayList<>();
+        for (String line : lines) {
+            Task task = Parser.parseTaskFromFile(line);
+            if (task != null) {
+                tasks.add(task);
+            }
+        }
+        return tasks;
+    }
+
+    public void save(TaskList tasks) throws IOException {
```

Comment:
```suggestion
    public void saveTasks(TaskList tasks) throws IOException {
```
Renaming might make it clearer that save saves many tasks